app/javascript/components/ArtistRequests.jsx

```diff
@@ -0,0 +1,226 @@
+      }
+      if (this.props.buyer) {
+        const buyer_id = this.props.buyer.id;
+        requests_route = APIRoutes.buyers.requests(buyer_id);
+      }
+      Requester.get(requests_route).then(
+        response => {
+          let responseWithWorks = response;
+          response.map((item, i) => {
+            Requester.get(APIRoutes.works.show(item.request.work_id)).then((data) => {
+              item.work = data;
+              this.setState({ inbox: this.state.inbox.concat(item), display: this.state.display.concat(i) });
+            });
+          });
+        },
+        error => {
+          console.error(error);
+        }
+      );
+    });
+  }
+
+  getAttr = (request) => {
+    console.log(request);
+    let attr = {
+      "Buyer": request.buyer.name,
+      "Artist": request.artist.name,
+      "Placed": new Date(request.request.updated_at).toLocaleDateString(),
+      "Request Type": request.request.types
+    };
+
+    if (!request.open && request.receipt) {
+      if (request.receipt.transaction_type === "rental") {
+        attr["Start Date"] = request.receipt.start_date;
+        attr["End Date"] = request.receipt.end_date;
+      }
+      attr["Price"] = request.receipt.price;
+      attr["Purchase Date"] = request.receipt.purchase_date;
+    }
+
+    return (
+      <div className="attr">
+        <div className="key">
+          {
+            Object.keys(attr).map((obj, i) => {
+              return <h5 key={i} className="attr-item">{obj}</h5>
+            })
+          }
+        </div>
+        <div className="value">
+          {
+            Object.keys(attr).map((obj, i) => {
+              return <h6 key={i} className="attr-item">{attr[obj]}</h6>
+            })
+          }
+        </div>
+      </div>
+    );
+  }
+
+  RequestListItem = (props) => {
+    const request = props.request;
+    const id = request.request.id;
+    const buyer = request.buyer;
+    const artist = request.artist;
+    const title = request.work.title;
+    const thumbnail_url = request.work.thumbnail ? request.work.thumbnail : "https://cdn0.iconfinder.com/data/icons/typicons-2/24/image-128.png";
+    const open = request.request.open;
+    const created_timestamps = new Date(request.request.created_at).toLocaleDateString();
+    const closed_timestamps = new Date(request.request.updated_at).toLocaleDateString();
+
+    return (
+      <div className="content-row">
+        <div className="request-content">
+          <div>
+            <img src={thumbnail_url} className="w4"/>
+          </div>
+          <div className = "items">
+            {this.getAttr(request)}
+          </div>
+        </div>
+        <div className="request-buttons w5">
+          {
+            request.request.open ? (
+              <div className="w-100">
+                <StyledModal title="MARK AS COMPLETE">
+                  <CreateTransaction
+                    artist={this.props.artist}
```

Comment:
I don't think passing this in is necessary anymore since we're passing in the request now

Reply:
sending the request is unnecessary

Reply:
I don't think we actually ever use the artist prop within CreateTransaction though? Since now the relationship is just that it belongs to a request. I might just be confused lol